clang/test/Driver/ps5-linker.c

```diff
@@ -67,19 +67,24 @@
 // CHECK-NO-EXE-NOT: "--unresolved-symbols
 // CHECK-NO-EXE-NOT: "-z"
 
-// Test that an appropriate linker script is supplied by the driver.
+// Test that an appropriate linker script is supplied by the driver, but can
+// be overridden with -T.
```

Comment:
NIT: -T/--script

Reply:
There's no `--script` on `clang`. I may not understand your comment.

Reply:
No you haven't. I just assumed that there was, appologies.